snow/consensus/snowman/poll/early_term_traversal.go

```diff
@@ -271,15 +263,13 @@ func (p *earlyTermPoll) String() string {
 }
 
 func aggregateVotesFromPrefixesAndIDs(transitiveVotesForPrefixes []int, transitiveVotes bag.Bag[ids.ID]) []int {
-	voteCountsForIDsOrPrefixes := make([]int, 0, len(transitiveVotesForPrefixes)+len(transitiveVotes.List()))
-
-	for _, id := range transitiveVotes.List() {
+	transitiveVoteIDs := transitiveVotes.List()
```

Comment:
`.List()` allocates the list, so we should only call it once rather than twice if possible.